data/json/recipes/weapon/mods.json

```diff
@@ -1,30 +1,4 @@
 [
-  {
-    "result": "improve_sights",
```

Comment:
Please add deleted recipe to https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/recipes/recipe_obsolete.json